util/trace/perf_tiled.py

```diff
@@ -0,0 +1,52 @@
+# Compare performance of tiled vs untiled layers on snitch/snax
+import subprocess
+import json
+import itertools
+"""
+Does the following:
+0. make -C target... clean-data
+1. make -C target... all LENGTH=... TILE_SIZE=...
+2. run bin/snitch_cluster.vlt something.elf
+3. make traces
+4. Process logs
+5. TODO store logs somewhere
```

Comment:
I suggest to dump it in the same logs directory that is generated when we build tests? The one in `/repo/target/snitch_cluter/logs` because it's also automatically part of the `.gitignore` that was previously present.